tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
Well, I'm not sure how to proceed. This step sends a keyboard shortcut that opens a new window. This step might not always work and there is a bug report for that.

How can I use assert_screen with multiple alternatives in this case?

If I have two needles, one will be the new window (this needle is checked already) and one will possibly be the main window. The latter will actually check that the main window is clearly visibly which means that the new one did not open.

However, this also depends on the time. If I just use assert_screen, the second needle might always match first if openqa is under load. I need to give it some waiting time beforehand and I fail to see the benefit over a simple check_screen. Can someone assist on that? Thanks.